src/dotnet-interactive.Tests/StdioConnectionTests.cs

```diff
@@ -40,11 +42,10 @@ protected override Task<IKernelConnector> CreateConnectorAsync()
 
         var connector = new StdIoKernelConnector(
             command.ToArray(),
+            rootProxyKernelLocalName: "rootProxy",
```

Comment:
Does `CreateConnectorAsync` actually need to be `Task`-returning anymore?

Reply:
Looks like all implementations use `return Task.FromResult<>` (even before the changes in the current PR). I'll go ahead and change the return type to `IKernelConnector` instead.